docs/challenge-000.md

```diff
@@ -1,9 +1,51 @@
-# Challenge 0: Set up your development environment with Dev Containers
+# Challenge 0: Set up your development environment within a development container
 
 ## Challenge Objective
+The goal for this challenge is to set up and verify your dev environment in a VS Code dev container.
 
 ## Brief Guidance
 
+### Install Prerequisites
+Ensure you have a Microsoft Windows 10 or MacOS powered workstation with the following installed:
+- [Docker Desktop for Windows/MacOS](https://www.docker.com/products/docker-desktop)
+- [Visual Studio Code](https://code.visualstudio.com/)
+- [Visual Studio Code - Remote Development Extension Pack](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack)
+- [Azure Storage Explorer](https://azure.microsoft.com/en-us/features/storage-explorer/)
+- [Storage emulator](https://docs.microsoft.com/en-us/azure/storage/common/storage-use-emulator) (Windows) or [create a storage account in Azure](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-portal) (Mac)
+- [Postman](https://www.postman.com/downloads/)
+
+### Set up a VS Code dev container
+
+We'll be using a [dev container](https://code.visualstudio.com/docs/remote/containers) for this OpenHack to seamlessly set up your environment. To get started, follow these steps:
+
+1. Install and configure Docker for your operating system.
+
+    Windows / macOS:
+    1. Install Docker Desktop for Windows/Mac.
+    2. If not using WSL2 on Windows, right-click on the Docker task bar item, select **Settings / Preferences** and update **Resources > File Sharing** with any locations your source code is kept. See [tips and tricks](https://code.visualstudio.com/docs/remote/troubleshooting#_container-tips) for troubleshooting.
```

Comment:
Maybe add a heading that this is Windows specific. 
For MacOs, things look much simpler.

Reply:
i'm not sure how to edit for macs if this isn't correct. i was following the docs here: https://code.visualstudio.com/docs/remote/containers